tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -663,16 +663,36 @@ public function provide70Cases()
     ClassG
 };
 use const some\a\{ConstA, ConstB, ConstC};
+use const some\b\{
+    ConstA,
+    ConstB,
+    ConstC
+};
 use function some\a\{fn_a, fn_b, fn_c};
+use function some\b\{
+    fn_a,
+    fn_b,
+    fn_c
+};
 ',
                 '<?php
 use some\a\{ClassA, ClassB, ClassC as C};
+use function some\b\{
+    fn_b,
+    fn_c,
+    fn_a
+};
 use function some\a\{fn_a, fn_b, fn_c};
 use A\B;
+use const some\b\{
+    ConstC,
+    ConstA,
+    ConstB
+};
 use const some\a\{ConstA, ConstB, ConstC};
 use some\b\{
-    ClassF,
```

Comment:
It's actually a reminiscent of the early stage of my experiments. The tests I've added to the bottom should cover this situation as well. It could be rolled back. But, yes, it doesn't break and it covers a bit more.

At first glance this particular test seems to focus on ordering namespaces vs. functions vs. constants vs. classes but there was already some imports out of order inside the one of the groups. So I'm not really sure of the organization logic there. 

I guess we could refactor the tests as to leave each testing exactly one behavior. I'm not sure this is necessary or desirable, though.

I'm up to whatever you guys decide.